packages/web3-react/src/components/ConnectModal/ConnectWithInjector/index.tsx

```diff
@@ -92,7 +92,7 @@ const ConnectWithInjector: React.FC<{
   const { connect } = useConnect({
     chainGroup: context.addressGroup,
     keyType: context.keyType,
-    networkId: context.network
+    networkId: context.network ?? ''
```

Comment:
Ah, there is already one in the CLI! Nice!

Reply:
I would suggest that we export an array with the valid values so that we can do validation when needed, and also generate a type out of the array (so that we only define the valid list of values once). Here's how we can do that:

```ts
export const networkIds = ['mainnet', 'testnet', 'devnet'] as const
export type NetworkId = (typeof networkIds)[number]
```

We can update the `NetworkType` type from the CLI tool to this and use it everywhere else. WDYT?

Reply:
I agree with what you said, but I think we should add this to @alephium/web3. I'm working on that now.

Reply:
That's a great idea 👍 We use similar trick in WalletConnectProvider